### PR TITLE
Update spanish language translation file

### PR DESCRIPTION
**Pull request by chuiizeet**
## What does this PR do?

Spanish translation.

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes.

### CODE REVIEW

File: app/config/locale/translations/es.json [hunk 1]
```diff
@@ -3,30 +3,30 @@
...
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Verificación de cuenta",
+    "emails.verification.hello": "Hola {{name}}",
+    "emails.verification.body": "Haz clic en este enlace para verificar tu correo.",
+    "emails.verification.footer": "Si no has solicitado verificar este correo, puedes ignorar este mensaje.",
+    "emails.verification.thanks": "Gracias",
+    "emails.verification.signature": "Equipo de {{project}}",
+    "emails.magicSession.subject": "Inicio de Sesión",
```
Comment (OscarRG) [inline app/config/locale/translations/es.json:12]: Hello, 'Sesión' should be written in lower case 'sesión'

File: app/config/locale/translations/es.json [hunk 2]
```diff
@@ -3,30 +3,30 @@
...
+    "emails.magicSession.footer": "Si no has solicitado ingresar usando este correo, puedes ignorar este mensaje.",
+    "emails.magicSession.thanks": "Gracias",
+    "emails.magicSession.signature": "Equipo de {{project}}",
+    "emails.recovery.subject": "Restablecer contraseña",
+    "emails.recovery.hello": "Hola {{name}}",
+    "emails.recovery.body": "Haz clic en este enlace para restablecer la contraseña de tu proyecto {{project}}.",
+    "emails.recovery.footer": "Si no has solicitado restablecer la contraseña, puedes ignorar este mensaje.",
+    "emails.recovery.thanks": "Gracias",
+    "emails.recovery.signature": "Equipo de {{project}}",
+    "emails.invitation.subject": "Invitacion al Equipo %s en %s",
```
Comment (OscarRG) [inline app/config/locale/translations/es.json:24]: Hi, 
'Invitacion' => you need the accent on the 'o'. 'Invitación'.
Equipo => should be written in lower case. 'equipo'

**Review by OscarRG — COMMENTED** [review]
Hi, 
i added some comments for typos :)

File: app/config/locale/translations/es.json (hunk 2 above)
Comment (chuiizeet) [reply app/config/locale/translations/es.json]: Hey, thanks a lot! Already fix that

**Review by christyjacob4 — APPROVED** [review]

**Comment by christyjacob4** [comment]
THANK YOU so much! All changes merged 🥳

Please complete the PR template and reach out to me on our [Discord server](https://appwrite.io/discord) if you would like to claim your Appwrite swag!

**Pull request merged**